### PR TITLE
Add ADR for authentication gem

### PR DESCRIPTION
**Pull request by ddippolito**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1583

## Changes in this PR:
Track the decision to adopt Devise as authentication gem

### CODE REVIEW

**Review by csutter — APPROVED** [review]

File: documentation/adr/0007_use_devise_for_authentication.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Teaching Vacancies will soon start to develop a new set of features which ultimately will allow job seekers to apply for jobs directly on the TVS website.
+
+This feature constitutes an important milestone for the service and will require job seekers users to create accounts to manage their job applications. Account functionality will permit teachers to initiate web sessions on TVS website as well as manage their account (password resets, email verification and account closing).
+
+## Options considered
+
+| Name      | Link                                     | Notes                                                                                    |
+| --------- | ---------------------------------------- | ---------------------------------------------------------------------------------------- |
+| Devise    | https://github.com/heartcombo/devise     | Widely adopted in the Ruby. Easily available documentation and features covering most TVS needs |
```
Comment (david-mears-dfe) [inline documentation/adr/0007_use_devise_for_authentication.md:29]: `Widely adopted in the Ruby.` Did you mean the Ruby world?

File: documentation/adr/0007_use_devise_for_authentication.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+| Clearance | https://github.com/thoughtbot/clearance  | Easy to integrate and configure, not as feature rich as Devise                           |
+| Authlogic | https://github.com/binarylogic/authlogic | Little support                                                                                       | 
+| Rodauth   | https://github.com/jeremyevans/rodauth   | Scarsely adopted                                                                                      |
```
Comment (david-mears-dfe) [inline documentation/adr/0007_use_devise_for_authentication.md:32]: Scarcely*

Comment (david-mears-dfe) [reply documentation/adr/0007_use_devise_for_authentication.md:32]: or rarely

File: documentation/adr/0007_use_devise_for_authentication.md (hunk 1 above)
Comment (ddippolito) [reply documentation/adr/0007_use_devise_for_authentication.md]: 👍 Copy-paste error: addressed in 23a59f400c5a164cc0a9b70fcad932a0c02707e6

File: documentation/adr/0007_use_devise_for_authentication.md (hunk 2 above)
Comment (ddippolito) [reply documentation/adr/0007_use_devise_for_authentication.md]: Done in 23a59f4

**Review by david-mears-dfe — APPROVED** [review]

**Pull request merged**